File: python/character_match.py
# Don't forget to run your tests!

from operator import is_
import string


def is_character_match(string1, string2):
	list1 = list(string1.lower())
	list2 = list(string2.lower())
	for i,j in enumerate(list1):
		try:
			index = list2.index(j)
		except: return False
		if j == list2[index]:
			list2.remove(list2[index])
			continue
		else:
			return False
	return True

	



is_character_match('charm', 'march')

File: js/characterMatch.js
exports.isCharacterMatch = function(string1, string2) {
    const list1 = string1.toLowerCase().split("")
    let list2 = string2.toLowerCase().split("")

    for (let i = 0; i < list1.length -1; i++) {
        // const search = list2.find(x => x === list1[i]);
        let splice = list2.indexOf(list1[i])
        if (splice >= -1){
            list2 = list2.splice(splice,splice)
        } else {
            return false;
        }
    }
    return true

};
